src/Categories/Category/Complete/Properties/Construction.agda

```diff
@@ -74,17 +75,17 @@ module _ (prods : AllProductsOf (o′ ⊔ ℓ′)) (equalizer : ∀ {A B} (f g :
         K⇒ : K.N C.⇒ src
         K⇒ = OP.⟨ (λ j → K.ψ (lower j)) ⟩
 
-        Keq : (i : ∃₂ J._⇒_) → ϕ⇒ i C.∘ K⇒ C.≈ ψ⇒ i C.∘ K⇒
-        Keq i@(A , B , f) = begin
-          ϕ⇒ i C.∘ K⇒    ≈⟨ OP.commute _ _ ⟩
-          K.ψ B          ≈˘⟨ K.commute _ ⟩
-          F₁ f C.∘ K.ψ A ≈˘⟨ pullʳ (OP.commute _ _) ⟩
-          ψ⇒ i C.∘ K⇒    ∎
+        Keq : {i : ∃₂ J._⇒_} → ϕ⇒ i C.∘ K⇒ C.≈ ψ⇒ i C.∘ K⇒
+        Keq = begin
+          _ C.∘ K⇒       ≈⟨ OP.commute ⟩
```

Comment:
I'm very happy to remove inferable things 'on the right' but less so on the left (no need to revert these here - just a general comment.) Stuff on the left serves as documentation.

Reply:
This change was at least in part justifying to myself that the arguments _could_ be inferred and hence were safe to remove